docs/getting_started.md

```diff
@@ -0,0 +1,5 @@
+# Developer Getting Started Guide
```

Comment:
I think I would prefer to consolidate documentation in one place. I.e. it is all in the README. Just my 2 cents.

Reply:
This makes sense - this was originally modeled after what ReportStream does, but agreed, I'm not sure it adds much value here for them to be separate

Reply:
Major preference for single stop documentation. To me splitting it out only makes it harder to actually _get_ started.